node-module-system/calculate.ts

```diff
@@ -0,0 +1,30 @@
+import { add } from './add.js';
+import { subtract } from './subtract.js';
+import { multiply } from './multiply.js';
+import { divide } from './divide.js';
+
+const mathOperation = process.argv[3];
+const firstNumber = process.argv[2];
+const secondNumber = process.argv[4];
+
+if (mathOperation === 'plus') {
+  console.log(
+    'result:',
+    add(parseFloat(firstNumber), parseFloat(secondNumber))
+  );
+} else if (mathOperation === 'minus') {
+  console.log(
+    'result:',
+    subtract(parseFloat(firstNumber), parseFloat(secondNumber))
+  );
+} else if (mathOperation === 'times') {
+  console.log(
+    'result:',
+    multiply(parseFloat(firstNumber), parseFloat(secondNumber))
+  );
+} else if (mathOperation === 'over') {
+  console.log(
+    'result:',
+    divide(parseFloat(firstNumber), parseFloat(secondNumber))
+  );
+}
```

Comment:
Nice solution. Some suggestions to improve the code:
- You have code duplication in how you are formatting the result. If you change your mind on how to do that, you will have a lot of places to fix.
- You have code duplication in your use of `parseFloat`. Can you get rid of it?
- It's probably best to use `Number(x)` instead of `parseFloat(x)`, but that's just a preference.
- There is no error handling. What if the user mis-types the operation?